llvm/include/llvm/CodeGen/ISDOpcodes.h

```diff
@@ -514,6 +514,9 @@ enum NodeType {
   /// separately rounded operations.
   FMAD,
 
+  /// FMULADD - Performs a * b + c, with, or without, intermediate rounding.
+  FMULADD,
```

Comment:
I don't understand why you would need this. It makes sense to have the fmuladd in the IR, but in the backend from the context you know the exact choice and should just go to it

Reply:
Oh...wasm has its own fmuladd. Sigh.